packages/core/lib/segments/segment.d.ts

```diff
@@ -64,6 +64,8 @@ declare class Segment {
   format(): string;
 
   toString(): string;
+
+  serialize(object?: Segment): string;
```

Comment:
```suggestion
  serialize(segment?: Segment): string;
```
Same as comment above